droidlet/lowlevel/robot_mover_utils.py

```diff
@@ -96,6 +96,35 @@ def get_move_target_for_point(base_pos, target, eps=0.5):
 
     return [targetx, targetz, yaw]
 
+def get_step_target_for_straightline_move(base_pos, target, step_size=0.1):
+    """
+    Heuristic to get step target of step_size for going to from base_pos to target
+    in a straight line. 
+    Args:
+        base_pos ([x,z,yaw]): robot base in canonical coords
+        target ([x,y,z]): point target in canonical coords
```

Comment:
[x, z, yaw] is a different format than [x, y, z], no? we should name these formats differently instead of calling both "canonical coords".